modules/preview-react/menu/stories/Menu.stories.mdx

```diff
@@ -0,0 +1,273 @@
+import {Meta} from '@storybook/addon-docs/blocks';
+import {Menu} from '@workday/canvas-kit-preview-react/menu';
+
+import {Basic} from './examples/Basic';
+import {ContextMenu} from './examples/ContextMenu';
+import {ControlButton} from './examples/ControlButton';
+import {CustomMenuItem} from './examples/CustomMenuItem';
+import {Icons} from './examples/Icons';
+import {ManyItems} from './examples/ManyItems';
+
+<Meta title="Preview/Menu/React" component={Menu} />
+
+# Canvas Kit Menu
+
+<a href="https://github.com/Workday/canvas-kit/tree/master/modules/preview-react/README.md">
+  <img src="https://img.shields.io/badge/PREVIEW-beta-blueviolet" alt="PREVIEW: Beta" />
+</a>
```

Comment:
Left the badge here...LMK if I should remove it 👍

Reply:
Good catch. There's some [discussion](https://github.com/Workday/canvas-kit/discussions/1131#discussioncomment-950500) on this. I removed it in my Breadcrumbs PR. I think probably remove it for now, and when we have a good way to display this on the docs site, we can add it back.